packages/neuron-ui/src/components/LockInfoDialog/index.tsx

```diff
@@ -82,25 +58,41 @@ const LockInfoDialog = ({ lockInfo, isMainnet, onDismiss }: LockInfoDialogProps)
   }
 
   return (
-    <dialog ref={dialogRef} className={styles.dialog} role="presentation" onClick={e => onDialogClicked(e)}>
-      <div className={styles.container}>
-        <h2 title={t('transaction.lock-script')} className={styles.title}>
-          {t('transaction.lock-script')}
-        </h2>
-        <div className={styles.lock}>
+    <Dialog
+      show={Boolean(lockInfo)}
+      contentClassName={styles.content}
+      title={t('lock-info-dialog.address-info')}
+      onCancel={onDismiss}
+      footer=""
+    >
+      <>
+        <div className={styles.title}>{t('transaction.lock-script')}</div>
+        <div className={styles.codeBlock}>
           <div
             className={styles.copyBtn}
-            onClick={handleCopy}
-            onKeyPress={e => (e.key === 'enter' ? handleCopy : undefined)}
+            onClick={() => handleCopy(rawLock)}
```

Comment:
What is the difference from get `rawLock` from context or from a parameter?

Reply:
> What is the difference from get `rawLock` from context or from a parameter?

Because there is still a `handleCopy(shortAddr)` below that needs to reuse the code.